.github/workflows/zombienet_polkadot.yml

```
@@ -0,0 +1,82 @@
+name: Zombienet Polkadot
+
+on:
+  push:
+    branches:
+      - master
+  pull_request:
+    types: [opened, synchronize, reopened, ready_for_review]
+  merge_group:
+concurrency:
+  group: ${{ github.workflow }}-${{ github.event.pull_request.number || github.ref }}
+  cancel-in-progress: true
+
+env:
+  RUN_IN_CONTAINER: 1
+  FF_DISABLE_UMASK_FOR_DOCKER_EXECUTOR: 1
+  LOCAL_DIR: "./cumulus/zombienet/tests"
```

Should not it be `LOCAL_DIR: "./polkadot/zombienet-sdk-tests/tests"`?

---

Yes, `LOCAL_DIR` is used in old test. Should be `./polkadot/zombienet_tests`.

```suggestion
  LOCAL_DIR: "polkadot/zombienet_tests"
```

Thx!